arm-sql/2014-04-01/swagger/replicationLinks.json

```diff
@@ -0,0 +1,393 @@
+            "type": "string",
+            "description": "The name of the Azure SQL Database that has the replication link to be failed over."
+          },
+          {
+            "name": "linkId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The id of the replication link to be failed over."
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content"
+          },
+          "202": {
+            "description": "Accepted"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/replicationLinks": {
+      "get": {
+        "tags": [
+          "DatabaseReplicationLinks"
+        ],
+        "operationId": "Databases_ListReplicationLinks",
+        "description": "Returns information about Azure SQL Database Replication Links.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "databaseName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Azure SQL Database to retrieve links for."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ReplicationLinkListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ReplicationLinkProperties": {
+      "properties": {
+        "partnerServer": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the Azure SQL Server hosting the partner Azure SQL Database."
+        },
+        "partnerDatabase": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the partner Azure SQL Database."
+        },
+        "partnerLocation": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The Azure Region of the partner Azure SQL Database."
+        },
+        "role": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The role of the Azure SQL Database in the replication link.",
+          "enum": [
+            "Primary",
+            "Secondary",
+            "NonReadableSecondary",
+            "Source",
+            "Copy"
+          ],
+          "x-ms-enum": {
+            "name": "ReplicationRole"
```

Comment:
#Resolved

---
In reply to: [85580621](https://github.com/Azure/azure-rest-api-specs/pull/643#discussion_r85580621) [](ancestors = 85580621,85475162)